### ISSUE TITLE
Easily mark an entry as "Favorite" entry

### ISSUE THREAD

**Issue by OLLI-S** [keepassxreboot/keepassxc#4518]
## Summary
Add an easy way (just one mouse click) to mark an entry as "Favorite" entry.

## Details
In KeePassXC I often search for the same entries, so it would be cool to define an entry as "Favorite" entry so I can find the entry quicker.
**Technically** here the tag (#508) "favorite" is added/removed, but in the UI users can do this with a star icon on the page "Entry" (when they edit an entry).

Example how this will look in the UI of KeePassXC  (first screen shows the "favorite" tag is set, second screen shows the favorite tag not set):

![image](https://user-images.githubusercontent.com/8037790/77860634-8edb4780-7210-11ea-888d-84d70301aaf6.png)

![image](https://user-images.githubusercontent.com/8037790/77860676-ad414300-7210-11ea-94dc-7cd182ccc573.png)

With this icon users can easily mark an entry as "Favorite" with just one mouse click,
They do not have to switch to an other page and add the text "Favorite" manually to the tags field.

**Comment by dotancohen** [comment]
Marking favorites is the one really useful feature that I feel is missing in KeepassXC. In my opinion, it should be based on the [tags featre](https://github.com/keepassxreboot/keepassxc/issues/508) which is already implemented in Keepass.
My workaround is to place two backticks ` \`\` ` in the first line of the Notes section of entries that I would tag "Favorite". Then, I can simply type two backticks into the search bar to quickly access these entries.

**Comment by tunbridgep** [comment]
These items should also be shown first in the list in the browser plugin

**Comment by droidmonkey** [comment]
Now that we have tags you can add a favorite tag and use that

**Comment by droidmonkey** [comment]
Re-opening since the spirit of this request is a one-click option to add the "Favorite" tag for the user.

**Comment by olee** [comment]
Showing favorite entries topmost in the browser plugin or yet better, even highlighting them / showing a start next to them or whatever would indeed be really nice.
Also it would be nice if the favorite section could then be added somewhere else than at the bottom where the other tags are - for example with a button in the top toolbar or above the root folder in the tree view.

**Comment by boardtc** [comment]
I'm happy for this to be closed, I think the tag works well. You can right-click an entry and select tag/ favourite, that's one click. Job done. Great work.

**Comment by thomasfrobieter** [comment]
> I'm happy for this to be closed, I think the tag works well. You can right-click an entry and select tag/ favourite, that's one click. Job done. Great work.

Yes, this part is fine, but I don't see any way to show the FAV tags right at the start of KepassXC! So for me this is kind of useless, you still have to open the FAV tags group manually every time.

**Comment by GenocideStomper** [comment]
Ideally I would love the top-most root folder to be filled with my favorite entries, and should link to the actual entry.

I think a "symlinking" functionality needs to be added. So that if either the symlink entry or the actual entry in a subfolder is edited, its reflected in the other.

Currently there's functionality to clone and "replace username and password with references" an entry, however it would be great to expand those references to the whole entry.

**Comment by tunbridgep** [comment]
The easiest thing would probably be to add a setting to "display tag on startup" with the ability to specify a tag. If there's nothing specified, it defaults to the expired entries tag as it does now.
